### PR TITLE
Add tenants controller tests

### PR DESCRIPTION
**Pull request by arufanov**
Add tenants controller tests

### CODE REVIEW

File: src/main/java/com/a5lab/tabr/domain/tenants/TenantServiceImpl.java [hunk 1]
```diff
@@ -18,11 +18,19 @@ public class TenantServiceImpl implements TenantService {
 
   @Override
   @Transactional(readOnly = true)
-  public Collection<Tenant> findAll() {
-    return tenantRepository.findAll();
+  public Collection<TenantDto> findAll() {
+    return null;
```
Comment (arufanov) [inline src/main/java/com/a5lab/tabr/domain/tenants/TenantServiceImpl.java:22]: @yurichervoniy 
* could you help me with map list.

Comment (yurichervoniy) [reply src/main/java/com/a5lab/tabr/domain/tenants/TenantServiceImpl.java:22]: We have map function in mapper to convert Lists
Also you can write something like that 
tenantRepository.findAll().stream().map(tenantMapper::toDto).collect(Collectors.toList());

Comment (arufanov) [reply src/main/java/com/a5lab/tabr/domain/tenants/TenantServiceImpl.java:22]: @yurichervoniy 
* thanks!

**Comment by arufanov** [comment]
@yurichervoniy 
* could you review?

**Pull request merged**